test/release/examples/primers/listOps.chpl

```diff
@@ -0,0 +1,223 @@
+// List Operations
+
+/*
+  .. default-domain:: chpl
+
+  The Chapel :mod:`List` module provides the implementation of the ``list``
+  type. Lists are useful for building up and iterating over a collection
+  of values in a structured manner.
+*/
+
+private use List;
+
+config const quiet: bool = false;
+
+/*
+  We'll start by declaring a list of ``int(64)`` and initializing it with the
+  values contained in the range ``1..8``.
+*/
+
+var lst1: list(int) = 1..8;
+
+writeln("List 1 after init: ", lst1);
+
+/*
+  The most common operation performed on a list is ``list.append()``. The
+  following code appends some integers to the end of our list.
+
+  .. note::
+
+    The ``list`` type guarantees that appending a value to the end of a list
+    will not invalidate references to existing values.
+
+    This is particularly useful in a parallel context. One task may append
+    values to the end of a list while others index over existing values
+    and use them in useful work.
+*/
+
+for i in 1..8 do
+  lst1.append(i);
+
+writeln("List 1 after appends: ", lst1);
+
+/*
+  If a list is intended to be used in parallel, then the ``parSafe`` field
+  must be set to ``true`` at initialization. 
+
+  Let's create a new list and ``list.append()`` values to it in parallel.
+*/
+
+var lst2: list(int, parSafe=true);
+
+coforall tid in 0..3 with (ref lst2) do
+  for i in 1..8 {
+    const elem = tid * 8 + i;
+    lst2.append(elem);
+  }
+
+/*
+  Tasks spawned in a coforall loop aren't guaranteed to execute in a fixed
+  order. The contents of `lst2` might be out of order even though our loop
+  size is small (only 4 tasks).
+
+  We can call ``list.sort()`` on our list to be on the safe side.
+*/
+
+if !quiet then
+  writeln("List 2 before sort: ", lst2);
+
+lst2.sort();
+
+writeln("List 2 sorted: ", lst2);
+
+/*
+  We can create another new list with values that are copied from `lst2`.
+*/
+
+var lst3 = lst2;
+
+/*
+  Before zippering `lst2` and `lst3` together, it would be good to vary their
```

Comment:
I find wrapping variable names in double back-ticks easier to follow for what's code and what's not, e.g.

> Before zippering _lst2_ and _lst3_ together

vs.

> Before zippering `lst2` and `lst3` together

However, we're not consistent about this across the docs, so totally optional. (if you do change, there's lots of instances of this, not just these)